src/v2/models/AppState.js

```diff
@@ -105,9 +105,33 @@ export default Model.extend({
   },
 
   shouldReRenderView (transformedResponse) {
+    _.mixin({
+      nestedOmit: function (obj, iteratee, context) {
+        var result = _.omit(obj, iteratee, context);
+
+        _.each(result, function (val, key) {
+          if (typeof(val) === 'object') {
+            result[key] = _.nestedOmit(val, iteratee, context);
+          }
+        });
+
+        return result;
+      }
+    });
+
     const previousRawState = this.has('idx') ? this.get('idx').rawIdxState : null;
-    const identicalResponse = _.isEqual(_.omit(transformedResponse.idx.rawIdxState, 'expiresAt'),
+    const identicalResponse = _.isEqual(_.nestedOmit(transformedResponse.idx.rawIdxState, ['expiresAt', 'refresh']),
+      _.nestedOmit(previousRawState, ['expiresAt', 'refresh']));
+    const noPollRefreshNeeded = _.isEqual(_.omit(transformedResponse.idx.rawIdxState, 'expiresAt'),
```

Comment:
i am not sure why this needed.. `noPollRefreshNeeded`

Reply:
The point of having `noPollRefreshNeeded` was to trigger the update polling interval event if **_only_** `refresh` in the IDX response changed. `if (identicalResponse && !noPollRefreshNeeded)` means only refresh value changed. 

If we use the "Forgot Password" flow and we poll for email as an example, that interval is a steady 4000ms. In that case `identicalResponse` and `noPollRefreshNeeded` are true -- then we don't need to enter the new event triggering logic.

Reply:
`noPollRefreshNeeded` is confusing, possible change the name to `isDynamicRefreshInterval`